<launch>
    <arg name="object_name" default="bottle"/>
    <arg name="topic_darknet" default="/darknet_ros/bounding_boxes"/> <!-- YOLO -->
    <node pkg="centroidtracker_ros" name="pub_centroidtracker_ros" type="pub_centroidtracker_ros.py" output="screen">
        <param name="object_name" value="$(arg object_name)"/>
        <param name="topic_darknet" value="$(arg topic_darknet)"/>
    </node>
</launch>